src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -53,10 +53,6 @@ void uct_rdmacm_cm_ep_error_cb(uct_rdmacm_cm_ep_t *cep,
                                uct_cm_remote_data_t *remote_data,
                                ucs_status_t status)
 {
-    if (cep->flags & UCT_RDMACM_CM_EP_FAILED) {
```

Comment:
so now we don't have check that uct_rdmacm_cm_handle_event_disconnected() is not calling disconnect callback more than once?

Reply:
the check is moved to the same place where the flag is set - uct_rdmacm_cm_ep_set_failed

Reply:
but uct_rdmacm_cm_handle_event_disconnected() also calls uct_rdmacm_cm_ep_error_cb()

Reply:
disconnected != failed, if the EP is got disconnect event but valid, user have to call disconnect as well, if the EP failed during wiruep, then no need to notify remote side.